pandas/core/dtypes/common.py

```diff
@@ -120,17 +120,26 @@ def is_object_dtype(arr_or_dtype):
 
 
 def is_sparse(arr):
-    """
-    Check whether an array-like is a pandas sparse array.
+    """Check whether an array-like is a pandas sparse array.
```

Comment:
I think that's not 100% correct, technically speaking. For what it's explained later, I think the function checks whether an array-like is a 1-D pandas sparse array.